site/docs/api.md

```diff
@@ -5,4 +5,4 @@ permalink: /docs/api
 title: API Documenation
 ---
 
-This guide provides web services available to integrate ManageIQ with external applications. It details the specification of the ManageIQ REST API, which is implemented as standard REST HTTP requests and responses of content type JSON.
+On the panel to the left, you will find a detailed overview of REST API, reference material, appendices, and plenty of examples. The guides provide information on how to integrate ManageIQ with external applications. It details the specification of the ManageIQ REST API, which is implemented as standard REST HTTP requests and responses of content type JSON.
```

Comment:
"On the panel to the left" doesn't make sense as this text is used in more than one view. For example, the main docs view:

![](http://screenshots.chrisarcand.com/3hzpp.jpg)

Just say "In this guide you will find a detailed overview of..."

Reply:
Cool thanks @chrisarcand will make that change.